README.md

```diff
@@ -372,6 +410,8 @@ writable.
 
 Always prefer `ReadonlyArray` over a regular `Array` unless it must be possible to modify the Array.
 
+Arrays should be typed as `Array<any>` rather than `any[]` for consistency.
```

Comment:
We shouldn't use the `any` type either, maybe the example could be `Array<unknown>` rather than `any`

Reply:
Agree, I think we should change it to `T`

```suggestion
Arrays should be typed as `Array<T>` rather than `T[]` for consistency.
```